messages/ja.json

```diff
@@ -0,0 +1,111 @@
+{
+  "admin/editor.search-result.title": "検索結果",
```

Comment:
same

Reply:
Not sure if we do, but since the localization team decided to translate this, I don't see any problem in keeping it.